streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/StreamsUncaughtExceptionHandlerIntegrationTest.java

```diff
@@ -192,7 +192,7 @@ private static class ShutdownProcessor<KIn, VIn, KOut, VOut> implements Processo
         }
 
         @Override
-        public void init(final ProcessorContext<KOut, VOut> context) {} {
+        public void init(final ProcessorContext<KOut, VOut> context) {
             this.context = context;
```

Comment:
Seem `context` is actually not used? It might be better to remove `context` member variable entirely?

Reply:
Thanks I removed the field and the assignment